docs/about/data-model/log.md

```diff
@@ -57,11 +44,62 @@ free to use whatever fields and shape you like. In places where Vector must
 operate on a field, Vector will default to the [default schema](#default-schema)
 and provide options to specify custom field names.
 
-### Nested Keys
+### Fields
+
+A "field" represents a [key](#keys)/[value](#values) pair and a `log` event is
+comprised of many fields. 
+
+#### Keys
+
+Keys are `string` representations of the field name.
+
+##### Special Characters
+
+As described in the [nested fields](#nested-fields) section, only `.` is
```

Comment:
I believe square brackets are also special for encoding arrays. /cc @LucioFranco

Reply:
Good catch. I updated this.